src/celeritas/ext/GeantPhysicsOptions.hh

```diff
@@ -76,27 +80,40 @@ enum class RelaxationSelection
  */
 struct GeantPhysicsOptions
 {
+    //// Physics list ////
+    // Gamma
     bool coulomb_scattering{false};
+    bool compton_scattering{true};
+    bool photoelectric{true};
     bool rayleigh_scattering{true};
-    bool eloss_fluctuation{true};
-    bool lpm{true};
-    bool integral_approach{true};
+    bool gamma_conversion{true};
     bool gamma_general{false};
 
-    BremsModelSelection brems{BremsModelSelection::all};
-    MscModelSelection msc{MscModelSelection::urban_extended};
+    // Electrons and positrons
+    bool ionization{true};
+    bool annihilation{true};
+    bool brems{true};
+    BremsModelSelection brems_selection{BremsModelSelection::all};
+    MscModelSelection msc{MscModelSelection::urban};
     RelaxationSelection relaxation{RelaxationSelection::none};
 
+    //// Physics options ////
     int em_bins_per_decade{7};
+    bool eloss_fluctuation{true};
+    bool lpm{true};
+    bool integral_approach{true};
+
     units::MevEnergy min_energy{0.1 * 1e-3};  // 0.1 keV
     units::MevEnergy max_energy{100 * 1e6};  // 100 TeV
     real_type linear_loss_limit{0.01};
     units::MevEnergy lowest_electron_energy{0.001};  // 1 keV
+
+    // Multiple scattering parameters
     real_type msc_range_factor{0.04};
     real_type msc_safety_factor{0.6};
     real_type msc_lambda_limit{0.1};  // 1 mm
-    bool apply_cuts{false};
 
+    bool apply_cuts{false};
```

Comment:
Maybe move this parameter next to `lowest_electron_energy` since it's related to that?